packages/api/src/trpc.ts

```diff
@@ -120,6 +120,16 @@ const isAuthed = t.middleware(async ({ next, ctx }) => {
     throw new TRPCError({ code: "UNAUTHORIZED" });
   }
 
+  await ctx.db.participant.upsert({
+    where: {
+      id: 1,
+    },
+    update: {},
+    create: {
+      name: "Test",
+    },
+  });
```

Comment:
does this make sense to you @RyanL123

Reply:
basically every user needs to have a linked participant, so for now we just create a dummy participant

Reply:
this does make our admin case a bit wonky though.....

Reply:
maybe we just have a separate Admin table, don't lump them in with users?

Reply:
What's the reasoning for separating participants with users if they're a 1-to-1 relationship? Why can we not just re-use the users model?

Reply:
They aren't a 1-to-1 relationship -- like there can be 2 parents and one participant

Reply:
mb, i think i did a 1-to-1 relationship between User and Participant because that's what i thought we wanted but I can change it so a participant can be linked to many users

Reply:
oh I missed that in the schema 🤦 -- pls make that change so that a Participant can have multiple users

Reply:
ok 👍